nsv13/code/modules/overmap/overmap.dm

```diff
@@ -462,6 +470,10 @@ Proc to spool up a new Z-level for a player ship and assign it a treadmill.
 		weapon_types[FIRE_MODE_BROADSIDE] = new/datum/ship_weapon/broadside(src)
 	if(plasma_caster)
 		weapon_types[FIRE_MODE_PHORON] = new/datum/ship_weapon/plasma_caster(src)
+	if(crail)
+		weapon_types[FIRE_MODE_RAILGUN] = new/datum/ship_weapon/railgun(src) //add coax rail datum
+	if(hrail)
```

Comment:
This like most special weapons gets applied by overriding the apply weapons proc.
EG:
![image](https://user-images.githubusercontent.com/45891563/231603313-ece21bd3-46a6-4d09-9218-29ddc4def13d.png)
![image](https://user-images.githubusercontent.com/45891563/231603350-003b8dc3-ebc5-4fc5-bd23-92e715fb5520.png)